guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -93,6 +93,17 @@ Therefore, do not delete, block, or change permissions of the user during the to
 +
 The scope of the JWTs is limited to the registration endpoints only and cannot be used anywhere else.
 
+[NOTE]
+====
+{Project} generates the registration command with parameters that search resources by ID.
+However, you can edit the registration command to search the following resources by name:
```

Comment:
```suggestion
However, you can edit the registration command to search the following resources by title:
```

We use `title` instead od `name`. For example:
Operating system `RHEL 9.5` is the title, and `RHEL` is the name.
With host groups, if the host group has a parent, the tile would be `Parent / Child group.`